trustymail/trustymail.py

```diff
@@ -328,12 +324,115 @@ def dmarc_scan(resolver, domain):
                 value = options.split('=')[1].strip()
                 tag_dict[tag] = value
 
+            if 'p' not in tag_dict:
+                tag_dict['p'] = 'none'
+            if 'sp' not in tag_dict:
+                tag_dict['sp'] = tag_dict['p']
+            if 'ri' not in tag_dict:
+                tag_dict['ri'] = 86400
+            if 'pct' not in tag_dict:
+                tag_dict['pct'] = 100
+            if 'adkim' not in tag_dict:
+                tag_dict['adkim'] = 'r'
+            if 'aspf'not in tag_dict:
+                tag_dict['aspf'] = 'r'
+            if 'fo' not in tag_dict:
+                tag_dict['fo'] = '0'
+            if 'rf' not in tag_dict:
+                tag_dict['rf'] = 'afrf'
+            if 'rua' not in tag_dict:
```

Comment:
Should `domain.dmarc_has_aggregate_uri` and `domain.dmarc_has_forensic_uri` be set to `False` here?

Also, I think `dmarc_has_forensic_uri` is actually called `dmarc_forensic_uri` in lines 71 and 189 of `Domain.py`.  The name `dmarc_has_forensic_uri` is better, I think, because it agrees with `dmarc_has_aggregate_uri`.

Reply:
Whoops, Yes!

And they should both should be `dmarc_has_forensic_uri`

Fixed.